test_othermlpackages/test_pytorch.py

```diff
@@ -84,24 +84,28 @@ def get_loss(self, y_pred, y_true, X, *args, **kwargs):
     return net
 
 
+@pytest.mark.skip(reason="external library errors because of new scikit-learn release")
```

Comment:
I don't think we should completely skip them. They should be skipped only if the latest sklearn is installed, with a nice comment that once they support it we put them back

Reply:
ooh I agree, makes sense, brb with a commit